cmd/util/ledger/migrations/account_status_migration.go

```diff
@@ -70,8 +70,12 @@ func (as *AccountStatusMigration) Migrate(payload []ledger.Payload) ([]ledger.Pa
 	newPayloads := make([]ledger.Payload, 0, len(payload))
 
 	for _, p := range payload {
-		owner := p.Key.KeyParts[0].Value
-		key := p.Key.KeyParts[2].Value
+		k, err := p.Key()
+		if err != nil {
+			return nil, err
+		}
+		owner := k.KeyParts[0].Value
+		key := k.KeyParts[2].Value
```

Comment:
Is there some inherent structure to KeyParts?  Maybe we can convert it to either a Struct or an Array to remove even more dynamic allocations?

Reply:
@SaveTheRbtz  

>Is there some inherent structure to KeyParts? Maybe we can convert it to either a Struct or an Array to remove even more dynamic allocations?

Ledger key is designed to be flexible, so it can contain variable number of KeyPart.  But we have plans to limit uses of Ledger key to migration and reports so that we can reduce number of heap allocs.

This PR eliminates ledger.Key from mtrie leaf nodes' payload.

The next PR related to this would be to eliminate ledger.Key created outside of migration and reporting which would reduce number of heap allocs caused by ledger.Key.